products/oneplus.md

```diff
@@ -0,0 +1,103 @@
+---
+title: OnePlus
+category: device
+tags: oneplus 1plus mobile-phone
+iconSlug: oneplus
+permalink: /oneplus
+versionCommand: "Settings -> About device"
+releasePolicyLink: https://community.oneplus.com/thread/1462181
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: Active Major Updates
+discontinuedColumn: true
+eolColumn: Security Updates
+customColumns:
+-   property: supportedAndroidVersions
+    position: after-release-column
+    label: Supported Android
+    description: Supported Android versions range
+    link: https://endoflife.date/android
```

Comment:
I get it, but it would be basically the same problem with pixel and fairphone: in the custom column we are talking about the support of their own OSes. Fairphone is running Fairphone OS and Pixel is running their own flavor of Android (that doesn't have a name AFAIK ?). Fairphone OS is not a big derivative from AOSP but Pixel OS definitely adds a lot over Android.
Moreover OxygenOS version wasn't always corresponding with Android so could be source of confusion here. 
![image](https://github.com/endoflife-date/endoflife.date/assets/6412153/eb1b4c6d-fb2a-4d9e-98fe-5a5ba53720a9)

IMHO it's either we change the column on all 3 pages or none of them.

Reply:
My preference would be to stick to OxygenOS. We're trying to [Be Helpful](https://github.com/endoflife-date/endoflife.date/wiki/Guiding-Principles#be-helpful) to the end-user. From their perspective, the page should answer "What is the highest version of my OS I can upgrade to". Using OxgenOS numbering clarifies the same, since that is what OnePlus uses in their settings screen, and upgrade popups, and documentation?

Google still calls Pixel updates as Android updates, that seems okay.

>  Moreover OxygenOS version wasn't always corresponding with Android so could be source of confusion here.

This is more the reason to document it as OxygenOS version numbers - users reading an Android version number might be confused as well. We can perhaps add an equivalence table/link.